tfhe/src/integer/server_key/radix_parallel/scalar_comparison.rs

```diff
@@ -372,7 +373,7 @@ impl ServerKey {
             match self.is_scalar_out_of_bounds(lhs, rhs) {
                 Some(std::cmp::Ordering::Greater) | Some(std::cmp::Ordering::Less) => {
                     // Scalar is not within bounds so it cannot be equal
-                    return self.create_trivial_radix(0, lhs.blocks().len());
+                    return BooleanBlock::new_unchecked(self.key.create_trivial(0));
```

Comment:
should we have a new_trivial ? so that we can do BooleanBlock::new_trivial(true)

Reply:
If think I already added it, its a matter of using it there